### DISCUSSION TITLE
Sidechain input only works for the first plugin in the chain (Beta 15)

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#1067] [Help] [answered]
Hello,

I just noticed that when sending sidechain audio to an AG plugin, the sidechain audio is only passed to the first plugin in the chain.
It would be nice to either send sidechain audio to the whole chain of plugins or to a selected plugin.
I'm using Cakewalk (Windows) and AG server also on a Windows machine.

![image](https://user-images.githubusercontent.com/68957086/196784380-9203158f-8ceb-49a6-8322-5897ea127c77.png)

**Comment by emack999** [comment]
It would be better for you to make a separate instance of AG for each additional plugin you want to send sidechain audio to.  The routing would be much more clear.

**Reply by jaacMusic** [reply]
Hello emack999,
I think if I were to add an additional instance of the AG plugin for sidechaining audio, it would introduce more latency as the audio would have to come back to the DAW from the first AG plugin and then send it back to the AG server on the secondary AG plugin for processing.

**Reply by emack999** [reply]
Only if you're looking to sidechain the F6 from the Q8 (Using your example above). You could instead sidechain the F6 from the same audio source you are using for the Q8, in a separate instance.

**Reply by jaacMusic** [reply]
No really. Having two separate AG plugins in a track means that audio has to come back from the first AG plugin back to the DAW before it can be sent back to the 2nd AG plugin.

**Comment by apohl79** [comment, marked as answer]
Maybe one of the plugins in your stack is resetting the SC channel. AG is just passing it through each plugin. But I think it could maintain a copy of the original channel input and pass it to each plugin in the chain. A selection would be possible as well. Will add a feature request for this. cheers.

**Reply by jaacMusic** [reply]
Hi Andreas,

I think the SC signal is broken when having multiple plugins and a plugin that doesn't support SC is inserted between plugins that do support SC. Having a way to select a plugin for SC would be great!
Thanks!
